### PR TITLE
Rephrase contributors text to reflect this month's two-release nature

### PR DESCRIPTION
**Pull request by bradcray**
Since 1.21 and 1.22 are quite similar, I'm combining their contributor sections for 1.22.

Also, add Souris Ash as a recent contributor.

### CODE REVIEW

**Comment by bradcray** [comment]
@souris-dev: Would you double-check that I've spelled and alphabetized your name correctly?

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -4,9 +4,10 @@ Chapel Contributors
 The following people have contributed to the implementation of the
 Chapel release:
 
-Contributors to the current release
------------------------------------
+Contributors to the Chapel 1.21 and 1.22 releases
+-------------------------------------------------
 * Ben Albrecht, [HPE]
+* Sourish Ash, invidividual contributor
```
Comment (souris-dev) [inline CONTRIBUTORS.md:10]: It is Souris Ash (there's no 'h' at the end of Souris). Thanks for adding me!

Comment (bradcray) [reply CONTRIBUTORS.md:10]: Argh, I'm glad I asked!  I don't know how I get these things so wrong.  :P

Comment (bradcray) [reply CONTRIBUTORS.md]: Should be fixed now (but feel free to triple-check me).

Comment (souris-dev) [reply CONTRIBUTORS.md]: Yeah, it's correct now. Happens a lot with my name, no issues :-). Thanks again!

**Pull request merged**